modules/installation-configure-proxy.adoc

```diff
@@ -218,7 +219,8 @@ endif::aws[]
 ifdef::vsphere,vmc[]
 You must include vCenter's IP address and the IP range that you use for its machines.
 endif::vsphere,vmc[]
-<4> If provided, the installation program generates a config map that is named `user-ca-bundle` in
+<4> Optional: The policy to add the `AdditionalTrustBundle` to the trusted certificate store of nodes. The allowed values are `Proxyonly` and `Always`. Use `Proxyonly` to add the `AdditionalTrustBundle` only when `http/https` proxy is configured, and `Always` to always add the `AdditionalTrustBundle``. The default value is `Proxyonly`.
```

Comment:
Should there be a "If provided" statement for policy to describe the behaviour? Like "If provided, the `Proxy` object is configured to reference the `user-ca-bundle` config map in the `trustedCA` field"

Reply:
Here `If provided` = `Optional:` But I can replace the existing `add the `AdditionalTrustBundle` to the trusted certificate store` to what you have suggested. Any thoughts on this?

Reply:
Sounds good to me, thanks

Reply:
I have incorporated your suggestions as well as changed the sequence of `additionalTrustBundlePolicy` as discussed